src/Native/Windows/clrcompression/Resources.rc

```diff
@@ -0,0 +1,41 @@
+#include "_Version.h"
```

Comment:
Actually it is already there it is called NativeVersion.rc is there some reason we cannot use that?